pineapple-bundle/pineapple-core/src/test/java/sh/miles/pineapple/util/serialization/bridges/YamlSerializedBridgeTest.java

```diff
@@ -54,7 +54,7 @@ void test_ShouldNotHaveAdapter() {
     @Test
     void test_ShouldDeserializeSerializeCorrectly() {
         test_ShouldRegisterAdapter();
-        ConfigWrapper wrapper = assertDoesNotThrow(() -> PineappleLib.getConfigurationManager().createDefault(new File(plugin.getDataFolder(), "config.yml"), YamlConfigMock.class));
+        Configuration wrapper = assertDoesNotThrow(() -> PineappleLib.getConfigurationManager().createConfiguration(new File(plugin.getDataFolder(), "config.yml"), YamlConfigMock.class).save(false).load());
```

Comment:
Should this be saving and loading in the does not throw? It replicates existing behaviour

Reply:
yeah it should